textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/FuzzyTokenizer.java

```diff
@@ -0,0 +1,38 @@
+package edu.uci.ics.textdb.dataflow.queryrewriter;
+
+/**
+ * Created by kishorenarendran on 25/04/16.
+ * Class to take in a string and uses a dictionary of words, to split it into possible
+ * set of strings, with appropriate space delimitions. his is a resource used by the QueryRewriter
+ * operator
+ *
+ *
+ * For example - "newyorkcity" - > ["new york city", "newyorkcity"]
+ * For example - "horseshoe" -> ["hor se shoe", "hors es hoe", "horse shoe", "horses hoe", "horseshoe"]
+ */
+public class FuzzyTokenizer {
```

Comment:
@chenlica : Since QueryRewriter handles multiple issues such as **fuzzy tokenization** (i.e. considering missing white spaces) as well as handling **synonym and acronym equivalence**, it made sense to have different modules for each functionality. Since for the first step, we are only considering the first functionality, we created a separate `FuzzyTokenizer` class. Later, we plan to also include a class for the second functionality.